City-of-Helsinki/helsinki-paatokset
# PP-251: Meetings and policymakers cleanup, part 2

This PR ports over functionality from paatokset_ahjo module to paatokset_ahjo_api and paatokset_policymakers, and switches over from Open Ahjo policymakers, and meetings and agenda items to Ahjo API policymakers and meetings.

**To test**
* Read code changes. Is there anything missing from the policymaker + meetings logic?
* Take a backup of your existing local environment with `drush make drush-create-dump`
* Run `make new`
* SSH into the container and run `drush ap:fs -v`
* Run the policymaker and meeting integrations: `drush mim ahjo_meetings:all; drush mim ahjo_decisionmakers:all`
* Check that policymakers were imported: https://helsinki-paatokset.docker.so/fi/admin/content/decisionmakers
* Check the kaupunginvaltuusto page. The heading should use the correct color coding
* Make sure meetings and documents are available: https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto
* Note one of the meetings on the page and add a new minutes of discussion document related to that meeting: https://helsinki-paatokset.docker.so/fi/media/add/minutes_of_the_discussion
  * The minutes PDF should now appear on the "Discussion minutes page"
* Click the meeting title on the discussion minutes page. It should lead to a page for that meeting which should have only finnish agenda points listed. The discussion minutes should be listed on this page also.
